pkg/cmd/server/kubernetes/master.go

```diff
@@ -126,6 +127,11 @@ func (c *MasterConfig) RunScheduler() {
 	glog.Infof("Started Kubernetes Scheduler")
 }
 
+func (c *MasterConfig) RunResourceQuotaManager() {
+	resourceQuotaManager := resourcequota.NewResourceQuotaManager(c.KubeClient)
+	resourceQuotaManager.Run(10 * time.Second)
```

Comment:
In current code, it affects two things at this time:

1. If you add a ResourceQuota to a project, you will not be able to add resources in that project until the synch has been run.
2. If you delete a resource, we only capture the delete until a synch has been run.  If you are running at the edge of your quota, you will need to wait.

Right now, upstream it runs at 10s, but it is configurable in kube-controller-manager.

I have a card in the future to make the delete case less time-sensitive, basic idea would be to watch for delete events on pods, and update the related pod.Namespace quota to release that memory/cpu since that is probably most important.